src/ledger/LedgerManagerImpl.cpp

```diff
@@ -388,11 +394,12 @@ LedgerManagerImpl::loadLastKnownLedger(bool restoreBucketlist)
     }
 
     // Step 4. Restore LedgerManager's internal state
```

Comment:
Nit: "Restore LedgerManager's LCL state" now that the LedgerManager technically stores different state variables.

Reply:
done